src/main/java/tusky/repository/FileManager.java

```diff
@@ -0,0 +1,106 @@
+package tusky.repository;
+
+import com.google.gson.ExclusionStrategy;
+import com.google.gson.FieldAttributes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import tusky.tasks.Task;
+import tusky.tasks.Event;
+import tusky.tasks.Deadline;
+import tusky.tasks.ToDo;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.ArrayList;
+public class FileManager {
+    private static final String DIR_NAME = "data";
+    private static final String FILE_NAME = "tusky.txt";
+
+    private static Gson gson = new Gson();
+
+    public static String getPath(){
+        return DIR_NAME + "/" + FILE_NAME;
+    }
+    public static ArrayList<Task> readFile () throws FileNotFoundException, NoSuchFileException {
+        File f = new File(getPath()); // create a File for the given file path
+        try {
+            Scanner s = new Scanner(f);
+//            Reader br = Files.newBufferedReader(Paths.get(getPath()));
+//            Task[] tasks = gson.fromJson(br, Task[].class);
```

Comment:
Remember to remove these lines of code if you are not planning to test it any further!